include/SFML/Graphics/Color.hpp

```diff
@@ -59,36 +59,36 @@ class SFML_GRAPHICS_API Color
     /// \param alpha Alpha (opacity) component (in the range [0, 255])
     ///
     ////////////////////////////////////////////////////////////
-    Color(Uint8 red, Uint8 green, Uint8 blue, Uint8 alpha = 255);
+    constexpr Color(Uint8 red, Uint8 green, Uint8 blue, Uint8 alpha = 255);
 
     ////////////////////////////////////////////////////////////
     /// \brief Construct the color from 32-bit unsigned integer
     ///
     /// \param color Number containing the RGBA components (in that order)
     ///
     ////////////////////////////////////////////////////////////
-    explicit Color(Uint32 color);
+    constexpr explicit Color(Uint32 color);
 
     ////////////////////////////////////////////////////////////
     /// \brief Retrieve the color as a 32-bit unsigned integer
     ///
     /// \return Color represented as a 32-bit unsigned integer
     ///
     ////////////////////////////////////////////////////////////
-    Uint32 toInteger() const;
+    [[nodiscard]] constexpr Uint32 toInteger() const;
```

Comment:
The "to" prefix implies a conversion; I don't see what the expectation of an expression like `color.toInteger();` would be. I would say this falls under the same as other getters, and thus doesn't need `[[nodiscard]]`.

Reply:
Fair enough, I will remove this one.